custom_components/myskoda/translations/en.json

```diff
@@ -121,6 +121,9 @@
         "number": {
             "charge_limit": {
                 "name": "Charge Limit"
+            },
+            "heater_duration": {
+                "name": "Heater Duration"
```

Comment:
Please be more specific here as well. I used to think this was enough, only to find out that:

1) Changing a translation key after introduction is troublesome since it becomes part of the entity id
2) Entities will get added that resemble yours